revscoring/extractors/api.py

```diff
@@ -239,6 +239,12 @@ def query_users_by_text(self, user_texts, batch=50, **params):
                                        ususers=batch_texts, **params)
 
                 for user_doc in doc['query'].get('users', []):
```

Comment:
Why not make global_user_info it's own thing?  Maybe having it be a subquery within user_info is unnecessary.